### ISSUE TITLE
Question (built using saved SQL question) not able to be filtered in Dashboard

### ISSUE THREAD

**Issue by cah-brandon-trebbien** [metabase/metabase#6310]
- Your browser and the version: Chrome Version 61.0.3163.100 (Official Build) (64-bit)
- Your operating system: Windows 10
- Your databases: Oracle 12
- Metabase version: 0.26.2
- Metabase hosting environment: Linux/RHEL 7
- Metabase internal database: MySQL 5.7

- *Repeatable steps to reproduce the issue*

1. Generate a freehand SQL based question which includes a SQL parameter. Ex. List of Facilities and Open Orders with the filter on Facilities. Add this question to dashboard 1. 
2. Generate a 2nd question, using the 1st question as a saved question. This question will aggregate the count of distinct orders by facility. Add this question to a dashboard 1.
3. Edit dashboard 1 and try to add a filter for facility. Only the freehand SQL based question shows as having the ability to be filtered. The 2nd question is not showing as being able to be filtered.

**Comment by btrebbien** [comment]
Update: I found this blog https://www.metabase.com/docs/latest/users-guide/13-sql-parameters.html

I completed the setup with my SQL having 
select *
from table
where 1 = 1
and {{variable}}

and set the variable type to Field Filter and linked it to the specific columns table. I refreshed the dashboard and can see the SQL question returning results as expected; however, clicking into the question from the dashboard and viewing the question itself it is returning oracle error ORA-00904: "WMLM"."ORDER_LINE_ITEM"."SHIP_WAVE_NBR": invalid identifier.  This is the exact full name of the schema.table.field in the oracle db.

For the original issue raised above, I can now filter the SQL based question in the dashboard (bullet step 1 above), but the 2nd question generated in step 2 above (aggregating the freehand SQL) is still not filterable using the {{variable}} setup in the SQL question.
